tools/nimbus-gradle-plugin/src/main/groovy/org/mozilla/appservices/tooling/nimbus/NimbusGradlePlugin.groovy

```diff
@@ -363,17 +363,19 @@ class NimbusPlugin implements Plugin<Project> {
             println args
         }
 
-        if(variant.hasProperty('registerJavaGeneratingTask')) {
+        if(variant.metaClass.respondsTo(variant, 'registerJavaGeneratingTask', Task, File)) {
```

Comment:
This line was not in the fenix version of the file, so this test has never worked.

Reply:
Yep 😬

Reply:
I have very much confirmed it does now though!